app/controllers/assignments_controller.rb

```diff
@@ -667,6 +667,9 @@ def destroy
     rescue ActiveRecord::DeleteRestrictionError
       flash_message(:error, I18n.t('assignments.assignment_has_groupings'))
       redirect_back fallback_location: { action: :edit, id: @assignment.id }
+    rescue StandardError => e
+      flash_message(:error, "Problem with deletion: #{e.message}")
```

Comment:
make sure to add a test case that covers these two lines of code

Also, make sure to internationalize this string